### DISCUSSION TITLE
kex unable to connect to socket: connection refused(10061)

### DISCUSSION THREAD

**Discussion by mzfr** [microsoft/WSL#6675] [Q&A] [answered]
I have installed `kali linux` using wsl2. Then I followed [this](https://www.kali.org/docs/wsl/win-kex/) guide to install `win-kex`. It was working fine till this morning.

Now if I try to run `kex` I get the following error:

```
Error connecting to the KeX server.
Please try "kex start" to start the service.
If the server fails to start, please try "kex kill" or restart your WSL2 session and try again.
```

If I run `kex stop` or `kex kill` I get the following output:

```bash
❯ kex stop                                                                                                                                                                                                     
tigervncserver: No matching VNC server running for this user!
tigervncserver: No matching VNC server running for this user!
```

```bash
~
❯ kex kill                                                                                                                                                                                                     tigervncserver: No matching VNC server running for this user!
tigervncserver: No matching VNC server running for this user!
tigervncserver: No matching VNC server running for this user!
tigervncserver: No matching VNC server running for this user!
```

I googled around to find various methods to fix this. Most of the articles/discussion suggest the following ways:

1) Install dbus-x11
    - Already installed still having the issue
2) Running the following command:
    ```
     -> kex stop
     -> cd /home
     -> kex
    ```
   -  This is not working for me
3) Kill vncserver and then start kex
	```
	-> vncserver --kill :*
	-> kex stop
	-> kex kill
	-> kex
	```
	- this gives the same error

In the 3rd method if I do all that and then start the vncserver by running:

```
vncserver
```

I get:

```
New Xtigervnc server 'pc.localdomain:1 (pwnbox)' on port 5901 for display :1.
Use xtigervncviewer -SecurityTypes VncAuth -passwd /home/.vnc/passwd :1 to connect to the VNC server.
```

And if I run `kex` after that I get error saying `No Matching security Type`

**Comment by KingHack-X** [comment]
I don't have and answer. I was just gonna say that it does the same for me

**Reply by mzfr** [reply]
This is not the `exact` fix of the problem but if you want to you can use `vcxsrv` to run the GUI.

Basically install `xfce4` and `xfce4-goodies` and `kali-desktop-xfce`. Also install `vcxsrv`(either download from Sourceforge or use choco). `vcxsrv` will be installed on your windows machine and not inside the kali VM.

Now run following command in kali terminal:

- if `wsl1` then `export DISPLAY=:0.0`
- if `wsl2` then ` export DISPLAY=$(grep -m 1 nameserver /etc/resolv.conf | awk '{print $2}'):0.0`

Once all this is done, run `xfce4-session`, if everything was done correctly you'll see kali GUI.

**Reply by matteothebrave** [reply]
I have to add something. i tried almost everything but no success.. i just opened Kali as administrator and worked. i hope it works for someone too..

**Reply by aioont** [reply]
> I have to add something. i tried almost everything but no success.. i just opened Kali as administrator and worked. i hope it works for someone too..

Yes that works for root user because both normal and root have separate .vnc ( configuration files )

**Comment by afonsofrancof** [comment, marked as answer]
After hours of searching to no avail, I have found that you need to:

1. do `sudo su`  (all the commands need to be performed by root user)  ;
2.  delete the symlink in /tmp/.X11-unix  ( just do `rm /tmp/.X11-unix`) ; 
3.  run `vncserver` (It will now be able to create the server) ;
4.  Run `kex` and everything should work.

**Reply by mzfr** [reply]
@afonsofrancof I deleted my kali wsl so I'll just mark your answer as the legit one.

Also, I ended up doing [what I suggested in the answer above](https://github.com/microsoft/WSL/discussions/6675#discussioncomment-485805). Using xfce4 was way better cause I was able to use the application inside kali popped out like any application in a normal setup. That helped in doing `alt+tab` stuff easily. I even gave my OSCP with that setup.

**Reply by CodeBunny09** [reply]
still not working

!!!!

**Reply by choicesrmade** [reply]
worked for me

**Reply by KiriTowaha** [reply]
@afonsofrancof It worked for me by using root. But I want to use `kex --win` (without root)
`kex --win` (not working)
`sudo kex -win`(work)
`kex -esm`(work)
I have reinstall the kali and wsl2 still like that.

**Reply by G3eKLAZy** [reply]
> After hours of searching to no avail, I have found that you need to:
> 
> 1. do `sudo su`  (all the commands need to be performed by root user)  ;
> 2. delete the symlink in /tmp/.X11-unix  ( just do `rm /tmp/.X11-unix`) ;
> 3. run `vncserver` (It will now be able to create the server) ;
> 4. Run `kex` and everything should work.

help me a lot thanks

**Reply by HermanRas** [reply]
Thanks this was the answer for me 2

**Reply by SajjadToomari** [reply]
it worked, thanks!

**Reply by gwd999** [reply]
nope. following these tips did not cut it ... still got the same error 
BUT what did help was restarting the distro

**Reply by michaeldrawe** [reply]
This worked for me.

**Reply by michaeldrawe** [reply]
Correction. This works as root only, but when Im not running WSL Kali Linux as root, I run the following command:

kex --sl -s

I see this information

Checking /home/michael/.cache/sessions/xfce4-session-172.18.0.1:0
Enabling SL mode
Desktop started after 1 seconds, enabling transparency now
/usr/bin/kex: line 310: 18596 Segmentation fault      dbux-launch --exit-with-session xfce4-session > /dev/null 2>&1

Then I don't see the Linux Toolbar that did show up when I ran the above command as root.

I'm sure many people would admire running kex as root, but most people would appreciate the security and not running it as sudo.

**Reply by MadobyPy** [reply]
**Thank you soooo much**, it worked for me

**Reply by ziruil3** [reply]
Can I ask what is the vnc authentication password?

**Reply by ziruil3** [reply]
@afonsofrancof

**Reply by afonsofrancof** [reply]
> @afonsofrancof

no idea, sorry. I dont use kex anymore

**Reply by ziruil3** [reply]
ok. I have found the answer. Thank you anyway.

**Reply by the-bst** [reply]
And if doesn't work anyway (case for me) (in sudo) : 
- go to ` cd /tmp` and do `rm X1-lock` or X2, X3, ... (need to remove all X-lock file so check in the directory before)
- next, go to ` cd .X11-unix` directory and remove all files (X1, X2 , ...)
- Finaly, run `vncserver` and `kex`
It works !

**Reply by AnupNarvekar** [reply]
This indeed works, thanks a ton!
Just note that you've to start kex henceforth via su mode ($`sudo su`  then $`kex` )  or ($`sudo kex`)

**Reply by priyansh2998** [reply]
> > @afonsofrancof
> 
> no idea, sorry. I dont use kex anymore

what are you using now?

**Reply by yodancristino** [reply]
PERFECT!

Just followed the instructions and it WOKRED! :D 
Thank you!

**Reply by pacive** [reply]
A few months later I had the same issue, but deleting the /tmp/.X11-unix directory failed because of "Read only filesystem". When running `mount` it listed (among others): `none on /tmp/.X11-unix type tmpfs (ro,relatime)`

So I tried running `# umount /tmp/.X11-unix` and after that the directory could be deleted and kex started working.

**Reply by alterkeyy** [reply]
you save my day ,THANK you very much!!

**Reply by cosminiq** [reply]
> A few months later I had the same issue, but deleting the /tmp/.X11-unix directory failed because of "Read only filesystem". When running `mount` it listed (among others): `none on /tmp/.X11-unix type tmpfs (ro,relatime)`
> 
> So I tried running `# umount /tmp/.X11-unix` and after that the directory could be deleted and kex started working.

Super god answer. Thank you verry much. You advice is verry god

**Reply by fesalm3ida** [reply]
Thank you, its woks !!

**Reply by marcio-loiola** [reply]
Hi guys, in my case, the output of vncserver after unmount the archive was: 
![image](https://user-images.githubusercontent.com/58188658/212493461-0381df48-6784-42bb-857f-8a87047f09ee.png)
 someone could help?

**Reply by marcio-loiola** [reply]
vncserver or this option return the same for me

**Reply by ghost** [reply]
> > A few months later I had the same issue, but deleting the /tmp/.X11-unix directory failed because of "Read only filesystem". When running `mount` it listed (among others): `none on /tmp/.X11-unix type tmpfs (ro,relatime)`
> > So I tried running `# umount /tmp/.X11-unix` and after that the directory could be deleted and kex started working.
> 
> Super god answer. Thank you verry much. You advice is verry god

This worked for me!! Thanks

**Reply by aioont** [reply]
https://github.com/aioont/kex

Read documentation to get overview on kex and win-kex issue.

Hope it someone :)

**Reply by liangyi0506** [reply]
Your advice is very helpful! Thanks!!!

**Reply by Paciupa** [reply]
I think there is a better solution [here](https://github.com/microsoft/WSL/issues/9303#issuecomment-1345615675), thanks to @elsaco.
Just `sudo mount -o remount,rw /tmp/.X11-unix`, because `/tmp/.X11-unix` is an important system file. [Here](https://github.com/microsoft/WSL/issues/9126) are some details why `/tmp/.X11-unix` is read-only in WSL in new versions by design.
So, for now I have just rolled back to the previous version because in new versions seamless mode is also broken

**Reply by briangullo** [reply]
Thanks @Paciupa , your solution worked for me.

**Reply by ahenzen** [reply]
yeo when i rm /tmp/.X11-unix it says can't rm because it's a directory then i did sudo rmdir /tmp/.X11-unix and it said it couldn't rm because device or recourse is busy any help?

**Reply by aioont** [reply]
If you encountered the "device or resource is busy" error, it means that some processes are still actively using resources within that directory, and therefore, it cannot be deleted. To resolve this, you need to find the processes that are using the X server and terminate them. Before doing so, be aware that this can cause your graphical user interface to stop functioning, and you may need to restart your system to restore it.

To identify processes using the X server, you can use the lsof command (list open files). Open a terminal and run:
`sudo lsof /tmp/.X11-unix`
This will show you a list of processes that have files open within the /tmp/.X11-unix directory. Once you identify these processes, you can decide whether it's safe to terminate them or not. To terminate a process, you can use the kill command followed by the process ID (PID).

`sudo kill <PID>`

**Reply by ahenzen** [reply]
Thaks a lot but when I do lsof /tmp/.X11-unix it just doesn't respond, and nothing pops up I assumed it was because nothing was using it but i tried to rmdir it again and it still said resource is busy.

**Reply by aioont** [reply]
Try
`rm -rf  /tmp/.X11-unix`

Or you can use this script https://github.com/aioont/kex

**Comment by Dhalbakken** [comment]
Adding to the solution found by @afonsofrancof 
I was able to recreate this. Following the logs from vncserver stating:
> _XSERVTransSocketUNIXCreateListener: mkdir(/tmp/.X11-unix) failed, errno = 11
> _XSERVTransMakeAllCOTSServerListeners: failed to create listener for unix
> 
> Xvnc TigerVNC 1.11.0 - built 2021-03-22 21:21
> Copyright (C) 1999-2020 TigerVNC Team and many others (see README.rst)
> See https://www.tigervnc.org for information on TigerVNC.
> Underlying X server release 12010000, The X.Org Foundation
> 
> 
> Sun Aug  1 21:05:56 2021
>  vncext:      VNC extension running!
>  vncext:      Listening for VNC connections on all interface(s), port 5901
>  vncext:      created VNC server for screen 0
> 
> Sun Aug  1 21:06:26 2021
>  ComparingUpdateTracker: 0 pixels in / 0 pixels out
>  ComparingUpdateTracker: (1:-nan ratio)
> _XSERVTransSocketUNIXCreateListener: mkdir(/tmp/.X11-unix) failed, errno = 11
> _XSERVTransMakeAllCOTSServerListeners: failed to create listener for unix

I haven't done extensive testing, but I believe the issue resides in the way that the Standard User and associated permissions are created during initial setup through WSL2. As best I can tell, the **_only_** way to launch a KeX environment right now is to do so as root. However, this creates another potential issue: The KeX GUI environment's login _will be root_. 

Attempting to launch KeX using the user you create during initial setup causes the recurring error quoted above until KeX times out and throws

> Error connecting to the KeX server.
> Please try "kex start" to start the service.
> If the server fails to start, please try "kex kill" or restart your WSL2 session and try again.

I recommend posting this discussion as a bug report to the Kali Community and/or appropriate KeX developers.

**Reply by absly** [reply]
I suppose that u are right. My kex worked very well util I updated the windows system to 11 or  the wsl core or something else. 
This bug still troubles me, because the root account cant support Gui well, many software don't allow root login. 
By the way, have you or someone else reported this bug?

**Comment by MartinOberhauser** [comment]
Hey, just for your information. I had the same problem, but what worked out for me is i checked my WSL version and i saw that i used
version 1. So i upgraded to version 2 and now it runs perfecty!

Cheers!

**Reply by KiriTowaha** [reply]
root or non-root ?

**Reply by MartinOberhauser** [reply]
You have to set the WSL Version in Windows of the VM.
e.g. wsl --set-version kali 2

**Reply by tiagofe** [reply]
Worked for me too. Thanks!

**Comment by sameerapr** [comment]
> I have installed `kali linux` using wsl2. Then I followed [this](https://www.kali.org/docs/wsl/win-kex/) guide to install `win-kex`. It was working fine till this morning.
> 
> Now if I try to run `kex` I get the following error:
> 
> ```
> Error connecting to the KeX server.
> Please try "kex start" to start the service.
> If the server fails to start, please try "kex kill" or restart your WSL2 session and try again.
> ```
> 
> If I run `kex stop` or `kex kill` I get the following output:
> 
> ```shell
> ❯ kex stop                                                                                                                                                                                                     
> tigervncserver: No matching VNC server running for this user!
> tigervncserver: No matching VNC server running for this user!
> ```
> 
> ```shell
> ~
> ❯ kex kill                                                                                                                                                                                                     tigervncserver: No matching VNC server running for this user!
> tigervncserver: No matching VNC server running for this user!
> tigervncserver: No matching VNC server running for this user!
> tigervncserver: No matching VNC server running for this user!
> ```
> 
> I googled around to find various methods to fix this. Most of the articles/discussion suggest the following ways:
> 
> 1. Install dbus-x11
>    
>    * Already installed still having the issue
> 2. Running the following command:
>    ```
>     -> kex stop
>     -> cd /home
>     -> kex
>    ```
>    
>    
>        
>          
>        
>    
>          
>        
>    
>        
>      
>    
>    * This is not working for me
> 3. Kill vncserver and then start kex
>    ```
>    -> vncserver --kill :*
>    -> kex stop
>    -> kex kill
>    -> kex
>    ```
>    
>    
>        
>          
>        
>    
>          
>        
>    
>        
>      
>    
>    * this gives the same error
> 
> In the 3rd method if I do all that and then start the vncserver by running:
> 
> ```
> vncserver
> ```
> 
> I get:
> 
> ```
> New Xtigervnc server 'pc.localdomain:1 (pwnbox)' on port 5901 for display :1.
> Use xtigervncviewer -SecurityTypes VncAuth -passwd /home/.vnc/passwd :1 to connect to the VNC server.
> ```
> 
> And if I run `kex` after that I get error saying `No Matching security Type`

How to No matching VNC server Problem working 100%
https://youtu.be/WCLTQ0MIP28

**Comment by zzkluck** [comment]
simply use 'sudo kex' instead of 'kex' seems to work well

**Reply by dl1rich** [reply]
This literally solved all of my issues, thank you zzkluck :)

**Comment by elnur0000** [comment]
I have a similar problem in my win 11,
while solution provided by @afonsofrancof  worked, there are still some problems 
kex command gives "Error connecting to the KeX server."
and vncserver gives "Segmentation fault"
sudo kex command now works perfectly, but I would prefer not using root for security reasons

**Comment by m-belotti** [comment]
I've resolved with "kex --esm --ip -sound", working well as non-root account.

**Reply by gwd999** [reply]
I really like that solution, also started having the same issues out of nowhere, or b/c of switching to WIN11, had totally forgotten about the RDP option. So will stick with that for now - thnx for reminding me of `esm`   
Gonna add a nice alias to my `.zshrc` and wait for updates on the issue, I guess.

**Reply by REFSZIN** [reply]
TRX MANNNNNNNNN <3

**Comment by aioont** [comment]
Let me know these work ?

It's properly working on kali linux rootless on android.

https://github.com/aioont/kex


If your error still exist,then try these

 cd ~/.vnc/
rm -rf *.log *.pid
cd /
> rm -rf /tmp/.X* /tmp/.xfsm* /tmp/.l2s* /tmp/.ICE-unix
vncserver -kill :*
vncserver -depth 24 -geometry 1920x1080 :1


//Removing log pid files in ~/.vnc directory
//Removing some files and directories under /tmp

**Comment by VagnerSilva** [comment]
I've resolved with...


Run the ps command if there is any process referring to kex finalize it

sudo rm -rf   /tmp/.X* (if '/tmp/.X11-unix' exists)
sudo mkdir /tmp/.X11-unix
sudo chmod 1777 /tmp/.X11-unix
sudo chown root /tmp/.X11-unix/
vncserver -localhost no

sudo nano ~/.vnc/xstartup
```batch
#!/bin/bash

PATH=/usr/bin:/usr/sbin
unset SESSION_MANAGER
unset DBUS_SESSION_BUS_ADDRESS
export XDG_SESSION_TYPE=x11
export GDK_BACKEND=x11
exec startxfce4
```
sudo chmod 777  ~/.vnc/xstartup
vncserver -localhost no
kex --win -s

**Reply by corrupted-bios** [reply]
Solved my issue, thanks, I just had to run `# umount /tmp/.X11-unix` first, as suggested by @pacive

**Reply by Joshpcbrrj** [reply]
I solver my problem with it. I hope that my pc restart the solution dont crash. Thank @VagnerSilva, and @chadmelancia and @pacive too. You are are wonderful.

**Reply by nannerpusser** [reply]
> I solver my problem with it. I hope that my pc restart the solution dont crash. Thank @VagnerSilva, and @chadmelancia and @pacive too. You are are wonderful.

Actually it's the WSL shutdown and restart that is going to wipe  it out. At least this is the state of systemd init (it's actually systemd-tmpfiles) in WSLg and versions of Ubuntu and Debian I have been able to use kex on perfectly until last major WSL update pushed by MS. They changed the way /tmp/ sockets are being handled, or at least how the /tmp/.X11-unix socket is treated at boot by systemd. It is wiped out no matter what and remounted as ro when it needs to be rw. The fix I was using was to just umount /tmp/.X11-unix and remount with rw. Very much like parent comment here suggests doing with chmod.

**Comment by SK-Infidel** [comment]
Ok, took some sleuthing, but there was a lot going on with this error. 

To be clear, the problem as defined is not being able to run kex as a NON-ADMIN. 
running it with sudo, or as root is NOT a solution. The error occurs when trying to start kex/xfce as a non administrator. 

Because that's what you're supposed to do. Because security matters. You don't wanna be messing around with some of these tools and unwittingly jack your entire box up because you owned yourself, and that stuff happens for many reasons. 
Been up for a long time, too much coffee, not enough coffee, yadda yadda. Being a regular user while using hacking tools is a very good idea, and it's a good idea anyway. 

I tried everything in these suggestions, and nothing worked. In fact some made it worse. 

There's a crazy interplay of xfce components at work here. DBUS, ICE, PolicyKit, TigerVNC, etc. etc. etc. 

What I noticed was that DBUS didn't seem to know anything about me. In the .xsessionerrors file in ~ I saw repeated messages akin to 

"dbus-update-activation-environment: systemd --user not found, ignoring --systemd argument"

Kind of beside the point, as wsl doesn't use systemd, but the "user not found" part of that message is very intersting. 

I needed to let it know how to find me... and that we're not in a systemd init environment. 

`dbus-update-activation-environment --all`

Checked the logs again, and I was getting a dbus session, which was progress, so I went to the next error in the list.

"_IceTransmkdir: Owner of /tmp/.ICE-unix should be set to root"
So I checked a full installation into an actual vm of ubuntu and I noticed that the file was owned by root:root and the mode was 1777

So I recreated it with the proper permissions
```
cd /tmp
mkdir .ICE-unix
sudo chown root:root .ICE-unix
sudo chmod 1777 .ICE-unix
```
After that, the ICE error was gone in the ~.xsessionerrors log, but I was seeing 

"(xfwm4:7394): Gtk-WARNING **: 18:06:13.137: cannot open display: wayland-0

(xfce4-panel:7411): libxfce4ui-WARNING **: 18:06:14.261: ICE I/O Error"

Shopped around for the ICE runtime directory, and after I found it, I noticed there was a lock file and a pid for wayland 0, so I deleted them. 

cd /mnt/wslg/runtime-dir
`rm -rf wayland*`

Then I ran kex --win -s, and boom. I have a stable desktop. 

I wrote a script to fix it. you'll have set it to executable and run it sudo. Not as root. Non-admin account.. sudo... 

```
#Clean up lock and pid files that exist in various places

rm -rf ~/.vnc/*.pid
rm -rf /tmp/*.X
rm -rf /tmp/.xfsm*
rm -rf /tmp/*.l2s
rm -rf /tmp/ssh*
rm -rf /tmp/.X10-unix/*
rm -rf /mnt/wslg/runtime-dir/wayland*

#Set permissions for .ICE-unix

mkdir /tmp/.ICE-unix
rm -rf /tmp/.ICE-unix/*
sudo chown root:root /tmp/.ICE-unix
chmod 1777 /tmp/.ICE-unix


#Fix DBUS

dbus-update-activation-environment --all

echo "That's it, we're all done here."
```

Then just run "kex --win -s" normally. No insecure localhost options, no geometry in vncserver first.. just run it how it's designed to run.

Hope this helps.

**Reply by P33J3** [reply]
Manually running the commands in the script worked for me in January 2023 (newbie); 

updated  rm -rf /tmp/.X11-unix/*

and used @cosminiq advice to umount file first.

Thank you people.

**Reply by branderage** [reply]
thanks, works great!

**Comment by ryankamfolt** [comment]
Very simple fix for WSL. Go to google and download VNCViewer. Install in Windows. Open and when it asks for server to connect to type: localhost:5901. Enter the password you setup earlier for KEX and bam you are in!

**Reply by flabeee** [reply]
Worked for me! Much appreciated!

**Comment by carl1l** [comment]
i find a way to solve this problem,but it will create another problem. i use `net stop winnat` and `net start winnat.` then i can use kex --win normally,but now my kali can not conncet the internet

**Comment by carl1l** [comment]
i think maybe i find the way to solve my new problem.i just Remove this option and then shutdown my windows system.then i did not appear this problem
![image](https://user-images.githubusercontent.com/74102133/163925036-88ee01fc-7119-4432-9cd1-4760e50afbd3.png)

**Comment by mrdragonblack** [comment]
please help me  i using kali linux in wsl2 in win11 after update my wsl2 win kex won't connect to win kex server all win kex broken kex ls no working, win kex wont'connect , win kex esm just blank screen last win kex not working please help me sorry for bad english  i have screenshot  
![Screenshot (153)](https://user-images.githubusercontent.com/118146225/207503407-79731a09-3a66-46f1-9a34-01d00117c574.png)
![Screenshot (154)](https://user-images.githubusercontent.com/118146225/207503447-11bf83e2-9dea-4be5-986b-964fd4d67fd5.png)
thank you for your help

**Reply by VagnerSilva** [reply]
Open de prompt as admin.
netsh winsock reset
restart you windows

In some cases it helps.
I hope it solves your problem.

**Reply by mrdragonblack** [reply]
not work  win kex wont connect to kex that way i hate wsl  wsl suck

**Reply by Paciupa** [reply]
For me everything worked out fine with previous version from Windows optional feature (not WSL 1, just older WSL 2).
And I was able to repair win mode Win-KeX with `sudo mount -o remount,rw /tmp/.X11-unix`, [thanks](https://github.com/microsoft/WSL/issues/9303#issuecomment-1345615675) @elasco, if you want to use the latest WSL version from the Store

**Reply by mistrysiddh** [reply]
```
sudo mount -o remount,rw /tmp/.X11-unix
```
I have to write this code everytime when i login into kali

**Reply by Paciupa** [reply]
You can just add this command in your ~/.bashrc file and it will be executed every time you login into. (To find out more search about ~/.bashrc)

**Reply by mistrysiddh** [reply]
i have already added to .zshrc which is my default terminal but i use this command to access it.
```bash
wsl kex --sl -s
```

**Comment by mrdragonblack** [comment]
Thank for your help if not working thats ok if not your help maybe i Will
quit from wsl linux  wsl is suck for Linux


Pada tanggal Rab, 14 Des 2022 13.25, Vagner Silva ***@***.***>
menulis:

**Comment by tkurskiev** [comment]
In my case, restarting computer helped.

**Comment by DurvalPaicheco** [comment]
Run without docker,
install using NPM 

###` npm install -g @soketi/soketi`

After start the service  --
### `soketi start`

Doc : https://docs.soketi.app/getting-started/installation/cli-installation
